startup_scripts/includes/__init__.py

```diff
@@ -0,0 +1,11 @@
+from ruamel.yaml import YAML
+from pathlib import Path
+
+
```

Comment:
Although there are other places where there are two empty lines, to me they're a waste of screen estate most of the time.